targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC/trng_api.c

```diff
@@ -0,0 +1,57 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "fsl_rng.h"
+#include "trng_api.h"
+
+#if defined(DEVICE_TRNG)
+
+void trng_init(trng_t *obj)
+{
+}
+
+void trng_free(trng_t *obj)
+{
+}
+
+int trng_get_bytes(trng_t *obj, uint8_t *output, size_t length, size_t *output_length)
```

Comment:
Please suppress unused argument compiler warning for ```obj```.

Reply:
@mmahadevan108 Could you address that comment, too, when you're fixing the zeroization below?